train/train.py

```diff
@@ -150,20 +164,137 @@ def train():
     print(f"Training on {device}")
 
     timestamp = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
-    session_dir = os.path.join(f"model_saves", f"training_{timestamp}")
+    session_dir = os.path.join("model_saves")
+
+    wandb.init(
+        project="MLVM-Project",
+        name=f"harmonic_training_{timestamp}",
```

Comment:
Mi turba questo nome, per due motivi:
- è molto lungo
- non sai che modello è

Lo modificherei in qualcosa del tipo "{R|C}NN-{dataora}"

Cioè è vero che il modello è scritto nei config, ma se a colpo d'occhio voglio sapere come sono andati i modelli di un tipo rispetto all'altro, è complicato capire chi è chi.